src/components/entries/Checkbox.js

```diff
@@ -71,6 +77,7 @@ function Checkbox(props) {
  * @param {Function} props.setValue
  * @param {Function} props.onFocus
  * @param {Function} props.onBlur
+ * @param {String|import('preact').Component} props.tooltip
```

Comment:
Usually primitive types are lowercase, cf. https://jsdoc.app/tags-type.html
```suggestion
 * @param {string|import('preact').Component} props.tooltip
```